pkg/testutils/cleanup.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Similarly to @JoelSpeed I also do wonder if it would be worth confining `testutils` in its own go module, to confine it from the rest of the e2e suites and avoid importing all of it everywhere.

Reply:
I have made it into its own module. I've had some issues getting unit tests and liters to run on both modules, but I got it to work.